app/javascript/components/applications/ApplicationsPage.js

```diff
@@ -20,12 +12,17 @@ const ApplicationsPageTable = ({ applications, fields }) => {
   )
 }
 
+const layout = {
+  pageHeader: {
+    title: 'Applications'
+  }
+}
+
 const ApplicationsPage = (props) => {
   return (
-    <div>
-      <ApplicationsPageHeader/>
+    <TableLayout {...layout}>
```

Comment:
Any specific reason you're de-structuring `layout`, rather than just passing `pageHeader` as a prop? Seems like it would be nice to try and use a consistent style.